pep-9999.rst

```diff
@@ -0,0 +1,670 @@
+  requires that either `license` or `license_file` is set when you upload a
+  package.
+
+- PHP Composer `composer.json <https://getcomposer.org/doc/04-schema.md#license>_`
+  is overall similar to the npm package.json, but license information is different.
+  They use an SPDX License identifiers or "proprietary".
+  The License field is either a single string that can use something
+  which resemble SPDX license expression syntax with "and" and "or" keywords;
+  or this is a list of strings if there is a choice of licenses (aka. "disjunctive").
+
+- NuGet `packages <https://docs.microsoft.com/en-us/nuget/reference/nuspec#licenseurl>_`
+  were using only a simple license URL and are now specifying to use an SPDX
+  License expressions and/or the path to a license file within the package.
+  The NuGet.org repository states that they only accepts license expressions
+  that are approved by the Open Source Initiative or the Free Software Foundation.
+
+- Golang has no provision for any metadata beside what is needed for dependencies.
+  Licensing is left to community package managers.
+
+- Dart/Flutter `spec <https://flutter.dev/docs/development/packages-and-plugins/developing-packages#adding-licenses-to-the-license-file>_`
+  recommends to use a single LICENSE file that should contain multiple
+  license texts each separated by a line with 80 hyphens.
+
+- JavaScript Bower `bower.json <https://github.com/bower/spec/blob/master/json.md#license>_`
+  license field is either a single string or a list of strings using either
+  SPDX license identifier or path/url to a license.
+
+- Cocoaposds `podspec <https://guides.cocoapods.org/syntax/podspec.html#license>_`
+  license is either a single string or a mapping with type, file an text keys.
+  This is mandatory unless there is a LICENSE/LICENCE fie provided.
+
+- Haskell `Cabal <http://hackage.haskell.org/package/Cabal-3.0.0.0/docs/Distribution-License.html>_`
+  specified a single string with a list of accepted licenses. And also provides
+  a `mapping between their license and SPDX identifiers
+  <http://hackage.haskell.org/package/Cabal-3.0.0.0/docs/Distribution-SPDX-LicenseReference.html#t:LicenseRef>_`.
+
+- Erlang/Elixir `mix/hex package <https://hex.pm/docs/publish>` specifies a
+  licenses field as a required list of license srtings. It is recommended to
+  use SPDX License identifier.
+
+- D lang `dub packages <https://dub.pm/package-format-json.html#licenses>_`
+  define their own list of license identifiers and their own license expression
+  syntax: both are very similar to SPDX conventions.
+
+- R Package `DESCRIPTION <https://cran.r-project.org/doc/manuals/r-release/R-exts.html#Licensing>_`
+  defines its own sophisticated license expression syntax and list of licenses.
+  R has a unique way to support specifiers for license versions such as `LGPL (>= 2.0, < 3)`
+  in its license expression syntax.
+
+
+Some convention used elsewhere
```

Comment:
```suggestion
Conventions used by other ecosystems
```

Reply:
This should also be in the "Appendix"

Reply:
@pradyunsg
> This should also be in the "Appendix"

Do you mean I should start an Appendix section for the "survey" parts?

Reply:
I used a new Appendix with https://github.com/pombredanne/spdx-pypi-pep/pull/2/commits/da5ea2708cca19f3dd75f01256e8e37d4bf8d7cc

Reply:
And used you proposed wording change too